src/grpc/server.py

```diff
@@ -0,0 +1,52 @@
+from src.db.database import Database
+from src.grpc.generated.subscription_pb2_grpc import SubscriptionServiceServicer
+import grpc
+from src.grpc.generated.subscription_pb2_grpc import add_SubscriptionServiceServicer_to_server
+from src.grpc.generated.subscription_pb2 import (
+    CreateSubscriptionRequest,
+    GetSubscriptionsRequest,
+    ChangeSubscriptionRequest,
+    DeleteSubscriptionRequest,
+    ActivateSubscriptionRequest,
+    DeactivateSubscriptionRequest, 
+)
+
+# gRPC service wrapper
+class SubscriptionService(SubscriptionServiceServicer):
+    def __init__(self, database: Database) -> None:
+        self.database = database
+    
+    async def CreateSubscription(self, request: CreateSubscriptionRequest, context: grpc.aio.ServicerContext):
```

Comment:
These should then call their appropriate data layer functions. Handle business errors like missing, broken input, or something else.